config/routes.rb

```diff
@@ -21,10 +21,16 @@
     end
   end
 
+  concern :statusable do
```

Comment:
I wonder if `status` could actually be `show`? That said, maybe that's best left to a future refactor.